manifests/charts/istiod-remote/files/gen-istiod-remote.yaml

```diff
@@ -676,7 +675,10 @@ metadata:
 webhooks:
   - name: sidecar-injector.istio.io
     clientConfig:
-      url: https://istiod.istio-system.svc:443/inject
+      service:
```

Comment:
interesting. Why not have that as default value and use URL in all cases ? Is there a benefit on using service ? 

(just curious - no need to change )

Reply:
not sure of benefit of using service... it is just what has been working and no reason to update it when it is working :). but it won't work when config is in a different cluster.